libstuff/SLog.cpp

```diff
@@ -38,3 +38,21 @@ void SLogStackTrace(int level) {
         }
     }
 }
+
+string addLogParams(const string& message, const map<string, string>& params) {
```

Comment:
Make this take `string&& message` and directly modify message and return it. Then we don't need to copy `message` on every call.

Reply:
👍 Updated.